package.json

```diff
@@ -65,7 +65,7 @@
     "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-jest": "^23.0.0",
     "husky": "^4.0.0",
-    "jest": "^25.0.0",
+    "jest": "^26.0.0",
```

Comment:
`jest` and `ts-jest` will need to maintain version parity, so we should bump that package too.

Reply:
In which case we will have to wait as the latest version of `ts-jest` is v25.4.0: https://www.npmjs.com/package/ts-jest.

Options seem to be:
- Close this PR and create our own to bump both `jest` and `ts-jest` to v26 once the latter releases its version of that.
- Keep this PR open (which might go stale) and wait until `ts-jest` releases v26 and update this existing PR.
- Merge this PR on the basis that we are currently encountering no issues (although may well do if changes are added that expose the incompatibility of the two versions).

Reply:
https://github.com/kulshekhar/ts-jest/issues/1595 `kulshekhar/ts-jest`: **Support Jest 26** (`ts-jest` issue opened 37 minutes ago).

There is quite a lot of recent activity on the `ts-jest` repo which suggests it will not be too long before v26 is released.

Reply:
The `ts-jest` maintainers are usually very responsive 😄  We could add `DO NOT MERGE` label for now and check in on it again at the end of the day?